### PR TITLE
Add mvnFlags to the github workflows,

### PR DESCRIPTION
**Pull request by agibsonccc**
## What changes were proposed in this pull request?
Add mvnFlags to the github workflows, remove the remote debugging (un…usable in matrix workflows)

## How was this patch tested?

Manually

## Quick checklist

The following checklist helps ensure your PR is complete:

- [X ] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [X ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [X] Created tests for any significant new code additions.
- [X ] Relevant tests for your changes are passing.

### CODE REVIEW

File: .github/workflows/build-deploy-android-arm32.yml [hunk 1]
```diff
@@ -30,8 +30,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-android-arm32.yml]: nit: ending brace is missing

File: .github/workflows/build-deploy-android-x86.yml [hunk 2]
```diff
@@ -31,8 +31,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-android-x86.yml]: nit: ) is missing

File: .github/workflows/build-deploy-android-x86_64.yml [hunk 3]
```diff
@@ -32,8 +32,8 @@ on:
         default: ossrh
 
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-android-x86_64.yml]: nit: ) is missing

File: .github/workflows/build-deploy-linux-arm32.yml [hunk 4]
```diff
@@ -31,8 +31,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-linux-arm32.yml]: ( is left open

File: .github/workflows/build-deploy-linux-arm64.yml [hunk 5]
```diff
@@ -32,8 +32,8 @@ on:
         required: false
         default:
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-linux-arm64.yml]: nit: )

File: .github/workflows/build-deploy-linux-cuda-11.0.yml [hunk 6]
```diff
@@ -31,8 +31,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-linux-cuda-11.0.yml]: nit: brace

File: .github/workflows/build-deploy-linux-cuda-11.2.yml [hunk 7]
```diff
@@ -32,8 +32,8 @@ on:
         default: ossrh
 
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-linux-cuda-11.2.yml]: nit:brace

File: .github/workflows/build-deploy-linux-x86_64.yml [hunk 8]
```diff
@@ -31,8 +31,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-linux-x86_64.yml]: nit:brace

File: .github/workflows/build-deploy-mac.yml [hunk 9]
```diff
@@ -37,8 +37,8 @@ on:
         required: false
         default: macos-10.15
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-mac.yml]: nit:brace

File: .github/workflows/build-deploy-windows-cuda-11.0.yml [hunk 10]
```diff
@@ -31,8 +31,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-windows-cuda-11.0.yml]: nit: brace

File: .github/workflows/build-deploy-windows.yml [hunk 11]
```diff
@@ -31,8 +31,8 @@ on:
         required: false
         default: ossrh
 
-      modules:
-        description: 'Modules to build'
+      mvnFlags:
+        description: "Extra maven flags (must escape input yourself if used"
```
Comment (quickwritereader) [inline .github/workflows/build-deploy-windows.yml]: nit:brace

**Review by quickwritereader — DISMISSED** [review]

**Comment by agibsonccc** [comment]
@quickwritereader  resolved thanks

**Review by treo — DISMISSED** [review]

**Pull request closed without merging**